server/services/agent_service.py

```
@@ -14,34 +13,30 @@
 APP_NAME = "ptso_outfit"
 
-# Ensure A2A_BASE_URL points to this server so the orchestrator's
-# RemoteA2aAgent sub-agents can reach the A2A routes served here.
-_port = os.environ.get("PORT", "8000")
-if not os.environ.get("A2A_BASE_URL"):
-    os.environ["A2A_BASE_URL"] = f"http://localhost:{_port}"
 
-
-def _get_orchestrator():
-    """Import and return the PTSO orchestrator agent.
+def _get_outfit_agent():
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_get_outfit_agent`

(ANN202)